ci-operator/step-registry/ipi/conf/vsphere/check/ipi-conf-vsphere-check-commands.sh

```diff
@@ -10,6 +10,11 @@ if [[ -z "${LEASED_RESOURCE}" ]]; then
 fi
 
 declare vsphere_datacenter
+
+# only used in zonal and vsphere environments with
+# multiple datacenters
+
+declare vsphere_datacenters
```

Comment:
Per the test job [log](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/49193/rehearse-49193-periodic-ci-openshift-release-master-nightly-4.15-e2e-vsphere-ovn-serial/1763118495857381376), we need some update.
```suggestion
declare -a vsphere_datacenters
vsphere_datacenters=()
```